scripts/base/frameworks/software/main.zeek

```diff
@@ -239,7 +239,10 @@ function parse(unparsed_version: string): Description
 	return [$version=v, $unparsed_version=unparsed_version, $name=alternate_names[software_name]];
 	}
 
+# A cache for the proxies that stores the result of parsing unparsed_version.
 global parse_cache: table[string] of Description &read_expire=65secs;
+# A suppression cache for the workers to prevent sending the same information to the proxies multiple times.
+global found_cache: set[Info] &create_expire=10mins;
```

Comment:
Nit: I wanted to suggest to make the the expiration time `const-redef`, but I guess that won't even work due to parsing-time-evaluates-before-redef-in-effect? :-(